homework/homework2/task1/curry.py

```diff
@@ -0,0 +1,36 @@
+from typing import Callable, Any
+
+
+def curry_explicit(function: Callable, arity: int) -> Callable:
```

Comment:
Please, try to fix your functions for this case:
```
f4 = curry_explicit(print, 4)
g4 = uncurry_explicit(f4, 4)
f2 = f4(10)(20)
g2 = uncurry_explicit(f2, 2)
print('---')
f4(5)(15)(25)(35)
f2(30)(40)
print('---')
g4(5, 15, 25, 35)
g2(30, 40)
```
The expected output in this case:
```
---
5 15 25 35
10 20 30 40
---
5 15 25 35
10 20 30 40
```
But in your case, you will have an error